.github/workflows/sync-translated-content.yml

```
@@ -33,25 +47,20 @@ jobs:
 
       - name: Install all yarn packages
         working-directory: ${{ github.workspace }}/mdn/content
-        run: |
-          yarn --frozen-lockfile
+        run: yarn --frozen-lockfile
 
       - name: Sync translated content
         env:
           CONTENT_ROOT: ${{ github.workspace }}/mdn/content/files
           CONTENT_TRANSLATED_ROOT: ${{ github.workspace }}/files
         working-directory: ${{ github.workspace }}/mdn/content
-        run: |
-          yarn content sync-translated-content
-
-      - name: Commit changes
-        # git commit will fail if there are no changes but that's okay!
-        continue-on-error: true
-        run: |
-          cd $GITHUB_WORKSPACE
-          git remote add upstream "https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
-          git config --local user.email "github-actions[bot]@users.noreply.github.com"
-          git config --local user.name "github-actions[bot]"
-          git commit -a -m "[CRON] sync translated content" && \
-          git pull --rebase upstream main && \
-          git push upstream main
+        run: yarn content sync-translated-content ${{ matrix.lang }}
+
+      - name: Create PR with sync for ${{ matrix.lang }}
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: "${{ matrix.lang }}: sync translated content"
+          branch: ${{ matrix.lang }}-content-sync
```

Use language as postfix, to keep branches together.

```suggestion
          branch: content-sync-${{ matrix.lang }}
```